.github/CODEOWNERS

```diff
@@ -511,6 +511,7 @@ pep-0653.rst  @markshannon
 pep-0654.rst  @1st1 @gvanrossum
 pep-0655.rst  @gvanrossum
 pep-0656.rst  @brettcannon
+pep-0657.rst  @pfmoore
```

Comment:
The PEP number 657 has now been taken - can you update this to the next free number? Might be worth waiting until @dholth has got the CLA sorted, so we don't have to keep changing.